### ISSUE TITLE
Try zenhub

### ISSUE THREAD

**Issue by ytremolet** [JCSDA/fv3gfs#2]
and see what it  can do

**Comment by danholdaway** [comment]
will do. fyi, you're posting to the jcsda rather than ucar gh so not everyone will see this

**Comment by ytremolet** [comment]
Yes, I know. I didn't expect anybody would see it. We are playing with it
to try it.
